packages/create-sitecore-jss/src/templates/nextjs-personalize/.env

```diff
@@ -1,3 +1,5 @@
 BOXEVER_CLIENT_KEY=
 BOXEVER_API=
 BOXEVER_TARGET_URL=
+BOXEVER_SCRIPT_URL=
+CDP_POS=
```

Comment:
Add original values of these variables to our internal doc. Mentioned in the PBI